sphinx/builders/devhelp.py

```diff
@@ -24,13 +24,7 @@
 try:
     import xml.etree.ElementTree as etree
 except ImportError:
-    try:
-        import lxml.etree as etree
-    except ImportError:
-        try:
-            import elementtree.ElementTree as etree
-        except ImportError:
-            import cElementTree as etree
+    import lxml.etree as etree
 
 try:
     import gzip
```

Comment:
https://docs.python.org/release/2.4/lib/someos.html

It seems optional package until Python 2.4. It became common standard library from 2.5. Importing gzip should be success now.